src/aria2p/utils.py

```diff
@@ -240,7 +240,7 @@ def load_configuration() -> Dict[str, Any]:
     config_dict["DEFAULT"] = toml.loads(default_config)
 
     # Check for configuration file
-    config_file_path = Path(XDG_CONFIG_HOME) / "aria2p" / "config.toml"
+    config_file_path = Path(BaseDirectory.save_config_path("aria2p")) / "config.toml"
```

Comment:
This is triggering `FileExistsError'. I think you should simply import `xdg_config_home` from `xdg` and use it here like previously.

Reply:
I totally missed the goal of aria2p aiming to be multiplatform 😅. Using `xdg` we are only considering systems following FreeDesktop standards. So [`appdirs`](https://github.com/ActiveState/appdirs) seems to be the suitable alternative for determining appropriate platform-specific config dirs.

Reply:
Oh well, I missed it as well :rofl: Good catch, thanks!